phpcs.xml.dist

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<ruleset name="experimental-features">
+    <description>Sniffs for the coding standards of the Experimental Features plugin</description>
+
+    <!--
+    Pass some flags to PHPCS:
+     p flag: Show progress of the run.
+     s flag: Show sniff codes in all reports.
+    -->
+    <arg value="ps" />
+
+    <!-- Strip the filepaths down to the relevant bit. -->
+    <arg name="basepath" value="./" />
+
+    <!-- Whenever possible, cache the scan results and re-use those for unchanged files on the next scan. -->
+    <arg name="cache"/>
+
+    <!-- Check up to 20 files simultaneously. -->
+    <arg name="parallel" value="20"/>
+
+    <!-- Only check the PHP files. -->
+    <arg name="extensions" value="php" />
+
+    <!-- Set severity to 1 to see everything that isn't effectively turned off. -->
+    <arg name="severity" value="1" />
+
+    <!-- Check all files in this directory and the directories below it. -->
+    <file>.</file>
+
+    <rule ref="WordPress" />
+    <rule ref="WordPress-VIP-Go" />
```

Comment:
The Alley-Interactive ruleset includes both of these plus our own preferences, like array syntax.

Reply:
👍